src/components/CancelButton/CancelButton.module.scss

```diff
@@ -0,0 +1,21 @@
+.cancel {
+  color: #386eb3;
+  font-family: Poppins;
```

Comment:
Again, a lot of these styles aren't necessary to declare here as done on a global level